xtext/org.icyphy.linguafranca/src/org/icyphy/generator/GeneratorBase.xtend

```diff
@@ -565,6 +571,8 @@ abstract class GeneratorBase extends AbstractLinguaFrancaValidator {
                             targetClockSync = clockSyncMethod.ON
                         }
                     }
+                    case "build":
+                        targetBuildCommand = param.value.literal.withoutQuotes
```

Comment:
Note that `value.literal` could be `null` (if no quotes have been used, in which case this will trigger a NPE).

Reply:
That's a good point. Will watch out for these kinds of errors...